Zend/zend_execute_API.c

```diff
@@ -851,6 +851,9 @@ int zend_call_function(zend_fcall_info *fci, zend_fcall_info_cache *fci_cache) /
 		if (EXPECTED((func->op_array.fn_flags & ZEND_ACC_GENERATOR) == 0)) {
 			zend_init_execute_data(call, &func->op_array, fci->retval);
 			zend_execute_ex(call);
+			if (UNEXPECTED(ZEND_CALL_INFO(call) & ZEND_CALL_CLOSURE)) {
+        			OBJ_RELEASE((zend_object*)func->op_array.prototype);
+			}
```

Comment:
I mean that the "clean" fix for this leak should be like this: https://gist.github.com/dstogov/fc22f2c54664ff3fc1d1